Medium/CountTrue.js

```diff
@@ -0,0 +1,15 @@
+//  Create a function which returns the number of true values there are in an array.
+
+
+CountTure=(arr)=>
```

Comment:
Declare function variable  as `const`

Reply:
done with the changes 
commit Id d3d7b0c23ba82bc6ca27e0400fdca5202d9d7567